docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/requirements.rst

```diff
@@ -34,8 +34,8 @@ Hardware
 
 -  Each Determined agent node should be configured with at least two Intel Broadwell or later CPU
    cores, 4GB of RAM, and 50GB of free disk space. If you are using GPUs, Nvidia GPUs with compute
-   capability 3.7 or greater are required. These include K80, P100, V100, A100, GTX 1080, GTX 1080
-   Ti, TITAN, and TITAN XP.
+   capability 6.0 or greater are required. These include P100, V100, A100, GTX 1080, GTX 1080 Ti,
```

Comment:
let's use this opportunity to include the popular cards?

```suggestion
   capability 6.0 or greater are required. These include P100, V100, A100, RTX 2080 Ti, RTX 3090,
```

Reply:
(proof: https://lookerstudio.google.com/u/0/reporting/5f632989-d726-4f1a-a57b-7e6968e977b5/page/p_ritz0jhqqc)